pubspec.yaml

```diff
@@ -1,14 +1,19 @@
 name: tiled
-version: 0.6.0
+version: 0.8.0
```

Comment:
why this bigger increase?

Reply:
and probably update the README according to the version

Reply:
Sorry...